pkgs/development/python-modules/chainer/default.nix

```diff
@@ -35,7 +30,7 @@ buildPythonPackage rec {
 
   # avoid gpu tests
   checkPhase = ''
-    pytest tests/chainer_tests/utils_tests -k 'not gpu and not cupy'
+    pytest tests/chainer_tests/utils_tests -k 'not gpu and not cupy and not ideep'
```

Comment:
Please use pytestCheckHook and disabledTests.

Reply:
I fix it. https://github.com/NixOS/nixpkgs/pull/111021/commits/c355637b4f182379eb9ffa3d9c492d3fb0b2dd74